packages/database/src/migrations/20220823234713-AddLesmisFavouriteDashboardTable-modifies-schema.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`
+    CREATE TABLE user_favourite_dashboard_item (
+      id TEXT PRIMARY KEY,
+      user_id TEXT NOT NULL,
+      dashboard_item_id TEXT NOT NULL,      
+      UNIQUE (user_id, dashboard_item_id),
```

Comment:
A existing record represents a user put the dashboard item in their favourite list. 

`UNIQUE` will make sure there is no duplicated records.